index.js

```diff
@@ -1,27 +1,28 @@
 const express = require("express");
-const bodyParser = require("body-parser");
 require("dotenv").config();
 const scrapeController = require("./controllers/webscrapeController");
-
-const app = express();
+const { getDateRange } = require("./models/dates");
 const mongoose = require("mongoose");
-const todoRoutes = require("./routes/todo.js");
 
-main()
-  .then((res) => console.log(res))
-  .catch((err) => console.log(err));
+//If no parameters are provided, by default, startDate is today and endDate is 7 days from today
+//If only one parameter is provided, getDateRange() by default sets that parameter as startDate and endDate 7 days from startDate
+const { startDate, endDate } = getDateRange("20230901", "20230905");
 
 async function main() {
   await mongoose.connect(process.env.DB);
-}
+  console.log("Connected to the database");
+  
+  const app = express();
 
-app.use(bodyParser.json());
-app.use(bodyParser.urlencoded({ extended: false }));
+  app.use("/api/todo", require("./routes/todo.js"));
 
-app.use("/api/todo", todoRoutes);
+  app.get("/espn/getGamesByDates", (req, res) => {
+    scrapeController.scrapeEspn(startDate, endDate, req, res);
+  });
```

Comment:
rethink this logic... how will users pass endDate if this api doesn't allow them to? 
getDateRange is hardcoding the endDate, which is not acceptable.
As I said in the previous comment, this should be driven by url parameters to the api call